### ISSUE TITLE
After manually updating to 1.6: internal server error

### ISSUE THREAD

**Issue by nielsdrost7** [InvoicePlane/InvoicePlane#884]
@aksysict I've opened that issue for you
![error](https://user-images.githubusercontent.com/83404331/201535104-13c94254-03e8-43ff-ab90-194ec75ce5e6.png)

> After that i change the php version from.7.4 to 8.1 on the hosting provider and i get only blank page. When i remove the new lines (dotenv) in index.php and set the old codes. Then it workes only dashboard. Invoices/clients/settings and other stuff does not work. And i dont get a error. Only page cannot be load warning on google chrome.
i removed the first line in ipconfig.php ->

> i turned on the debug mode on google chrome see the screenshot attached. this is exact the same what happend with the code in ipconfig.php so nothing changes

**Comment by nielsdrost7** [comment]
@aksysict I took a look at your screenshot and it shows an internal server error (500)
Can you try and run `composer install` or `,composer update`

The internal server error would end up in your CodeIgniter log (hopefully) or in your server error log

**Comment by aksysict** [comment]
@nielsdrost7 thanks for the reply. it is not our server. it is a ftp of our webhosting how i can run these commands?

**Comment by nielsdrost7** [comment]
You run these commands by logging into the server, on the command line of that server.
Remember what I said about slack? This is what I meant.

- You need to install your site locally. Use either something like Xampp or Wampp or Mampp or use something like docker.

**Comment by adamedia-de** [comment]
Hi, did you already figure out, what happened?  Is the issue resolved? Also tried to update manually to 1.6 v3, replacing all files (except ipconfig.php and uploads) and unfortunately facing similar issues - no. Clean new installation of 1.6. v3 also failes on php 8.1 -  any help would be appreciated...

**Comment by nielsdrost7** [comment]
@adamedia-de remove the top line from ipconfig.php and see if you still have the same problem.

If it fails for you, I need screenshots and codeigniter log.

In the case of the issue, that I created, I did the following:
- remove that top line in ipconfig.php (about direct access)
- remove composer.lock
- run `composer install`

I'm helping users out on slack, since it's a lot of steps you need to check.

**Comment by Maybach91** [comment]
I got the following error:

```log
Uncaught Dotenv\Exception\InvalidFileException: Failed to parse dotenv file
```

And the instructions from @nielsdrost7 solved my error, for a clean install on a php@8.1 server. Thank you!

**Comment by nielsdrost7** [comment]
We've addressed above issue in the documentation that will be available on the wiki page:
- The ipconfig.php file contains a line in the top of the file, which the phpdotenv doesn't like
- Solution is to **rename** or **move** the `ipconfig.php` file (see documentation, soon)
- If you rename your `ipconfig.php` file you need to **adjust** your `index.php` to that new name (see documentation)
- Our tip is to rename your `ipconfig.php` file to _something only you know_, so visitors cannot 'guess' your new config file name.
- Don't forget to **adjust** your `index.php` file, it's **looking for your config file** in one of the first lines of code.
- After you've renamed your `ipconfig.php` file and **adjusted** your `index.php` file you need to **protect** your newly renamed file
